package.json

```diff
@@ -25,7 +25,7 @@
     "interstellar-sessions": "~0.0.10",
     "interstellar-ui-messages": "~0.0.3",
     "lodash": "^3.10.1",
-    "stellar-sdk": "^0.14.0"
+    "stellar-sdk": "^2.0.0-rc.2"
```

Comment:
Are you sure there were no breaking changes between these 2 version? Other than that 👍 .

Reply:
Confirmed that balances + transactions work locally, afaik the breaking changes were in types